app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java

```diff
@@ -145,6 +148,7 @@ public boolean onQueryTextSubmit(String query) {
     @Override
     public boolean onQueryTextChange(String query) {
         if (!TextUtils.isEmpty(query)) {
+            query = query + "<->" + trackId;
```

Comment:
We can only send a single string as a constraint to `Filter` so I concatenated `searchtext` and `trackId`.
`<->` acts as a delimiter so I can get the searchtext and trackId back by `split()`.